pkg/network/setup/network.go

```diff
@@ -107,8 +107,8 @@ func (v VMNetworkConfigurator) getPhase2NICs(domain *api.Domain, networks []v1.N
 			return nil, fmt.Errorf("no iface matching with network %s", networks[i].Name)
 		}
 
-		// Binding plugin and SR-IOV devices are not part of the phases
-		if iface.Binding != nil || iface.SRIOV != nil {
+		// Binding plugin, SR-IOV and Slirp devices are not part of the phases
+		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
```

Comment:
This change seems to fir the next commit.

Reply:
This change is done in this commit because it includes Slirp plugin sidecar injection.

Reply:
The SLIRP implementation in the sidecar should be able to work with this per my understanding. If the domain already has the configuration, it should do nothing.

I is hard to reason skipping the process here but at the same time not removing the code itself. I guess it is fine if there is no other choice, but it seems in this case there is.

Reply:
The code is removed in the next commit.
I separated the code removal to another commit so it would be easier to review, I can squash them.